daprdocs/content/en/operations/configuration/use-custom-certificates.md

```diff
@@ -80,7 +80,7 @@ spec:
 ...
 ```
 
-**Note for Windows containers:** When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.
+> ℹ️ **Note for Windows containers:** When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}}
When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.
{{% /alert %}}
```

See example note in https://docs.dapr.io/operations/components/component-scopes/

Reply:
Nice, switching.